src/table/context/TableContext.tsx

```diff
@@ -22,7 +22,7 @@ export const TableContextProvider = ({
   cellCoordMock,
   selectionDispatchMock,
 }: ContextProviderProps) => {
-  const [headRowHeight, setHeadRowHeight] = useState();
+  const [headRowHeight, setHeadRowHeight] = useState<number>(0);
```

Comment:
fyi it implies number since you are giving it a initial value

Reply:
Right, fixed